x-pack/filebeat/module/zeek/README.md

```diff
@@ -0,0 +1,74 @@
+# Zeek (Bro) module
```

Comment:
I kind of like the idea to have a dev readme in a module. I don't think we had something like this so far 👍